src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/HttpConnectionPoolManager.cs

```diff
@@ -258,14 +258,16 @@ public Task<HttpResponseMessage> SendAsync(HttpRequestMessage request, bool doRe
                     proxyUri = _proxy.GetProxy(request.RequestUri);
                 }
             }
-            catch (Exception)
+            catch (Exception ex)
             {
                 // Eat any exception from the IWebProxy and just treat it as no proxy.
                 // This matches the behavior of other handlers.
+                if (NetEventSource.IsEnabled) NetEventSource.Error(this, ex);
```

Comment:
Maybe instead of `ex` it could be `Exception from IWebProxy: {ex}` or something like that?

Reply:
Fixed.